src/validation.h

```diff
@@ -211,7 +211,6 @@ struct PackageMempoolAcceptResult
  * Try to add a transaction to the mempool. This is an internal function and is exposed only for testing.
  * Client code should use ChainstateManager::ProcessTransaction()
  *
- * @param[in]  pool               Reference to the node's mempool.
  * @param[in]  active_chainstate  Reference to the active chainstate.
  * @param[in]  tx                 The transaction to submit for mempool acceptance.
  * @param[in]  accept_time        The timestamp for adding the transaction to the mempool. Usually
```

Comment:
maybe remove the sentence that starts with "usually"? https://github.com/bitcoin/bitcoin/pull/23437#discussion_r760377136

Reply:
Done in 3f7d1c1